build.py

```diff
@@ -467,8 +467,11 @@
 # TODO: Probably this should be another 'product', but for now it's simply a phase
 foundation_tests = SwiftExecutable('TestFoundation', [
 	'TestFoundation/main.swift',
+        'TestFoundation/HTTPServer.swift',
```

Comment:
You're right, the other test files are found via the glob. I guess this is ok. I'm going to kick off at est.